cedar-policy-validator/src/human_schema/grammar.lalrpop

```diff
@@ -280,11 +280,12 @@ Idents: Vec<Node<Id>> = {
 }
 
 // Names := Name {',' Name}
-Names: Vec<Node<SmolStr>> = {
-    <n:Name> => vec![n],
+Names: NonEmpty<Node<SmolStr>> = {
+    <n:Name> => NonEmpty::new(n),
     <mut ns:(<Name> ",")+> <n:Name> => {
-        ns.push(n);
-        ns
+        let mut all = NonEmpty::new(n);
+        all.append(&mut ns);
+        all
```

Comment:
What is the impact of this change?

Reply:
A vec that was always non-emtpy is now represented by a `NonEmpty`. (A name can't be an empty sequence of ids)